lib/slurp/logs/subscriptions.ex

```diff
@@ -77,4 +79,20 @@ defmodule Slurp.Logs.Subscriptions do
   def put(log_subscription, store_id \\ Logs.LogSubscriptionStore.default_store_id()) do
     Logs.LogSubscriptionStore.put(log_subscription, store_id)
   end
+
+  defp parse_event_signature(signature) when is_bitstring(signature), do: parse_event_signature({signature, []})
+  defp parse_event_signature({event_header, address}), do: {event_header, address}
+
+  defp make_log_subscription(%{address: []} = attrs) do
+    make_log_subscription(%{attrs | address: ""})
+  end
+  defp make_log_subscription(%{address: [address | []]} = attrs) do
+    make_log_subscription(%{attrs | address: address})
+  end
+  defp make_log_subscription(%{address: [address | rest]} = attrs) do
+    make_log_subscription(%{attrs | address: address}) ++ make_log_subscription(%{attrs | address: rest})
+  end
+  defp make_log_subscription(%{address: address} = attrs) when is_bitstring(address) do
+    [struct!(Logs.LogSubscription, attrs)]
+  end
```

Comment:
I dont see many patterns like this in your project, if you disagree with this, let me know

Reply:
This lgtm

Reply:
I like it :+1: